consoleApp/src/main/java/com/issoft/store/StoreApp.java

```diff
@@ -42,13 +42,43 @@ else if(userCommand.equalsIgnoreCase("top")){
         else if(userCommand.equalsIgnoreCase("exit")) {
                 reader.close();
                 System.out.println("See you soon!");
-                i = 0;
+                System.exit(0);
             }
             else {
                 System.out.println("The command was not recognized. Available commands: sort, top, exit. Please try again.");
             }
+
+            final Thread purchaseGoods = new Thread(new Runnable() {
+                @Override
+                public void run() {
+                    int i = new Random().nextInt(30);
+                    try {
+                        Thread.sleep(100*i);
+                    } catch (InterruptedException e) {
+                        e.printStackTrace();
+                    }
+                    List<Product> purchasedGoods = randomStorePopulator.createOrder(extractedProducts);
```

Comment:
Only one product from extractedProducts should be added to purchasedGoods collection (which will be selected by user)